### DISCUSSION TITLE
blog/201113-vscode-cmake/

### DISCUSSION THREAD

**Discussion by giscus** [evandde/blog_comment#35] [Announcements]
# blog/201113-vscode-cmake/

VSCode에서 CMake 기반의 프로젝트를 만들고, 실행/디버깅 등이 가능한 환경을 구축하는 방법에 대해 알아봅니다.

https://evanote.mew.kr/blog/201113-vscode-cmake/

**Comment by tobony** [comment]
감사합니다
